composer.json

```diff
@@ -14,7 +14,7 @@
         "doctrine/common": "^3.1",
         "doctrine/dbal": "^2.13 || ^3.0",
         "doctrine/event-manager": "^1.1",
-        "doctrine/orm": "^2.10",
+        "doctrine/orm": "^2.11",
```

Comment:
IMO, this change is not related to the described fix.

Reply:
2.11 added the `enumType` feature for the mapping which is needed to reproduce the issue (and is used in the test) but I see now that you test PHP 8.1 only against `highest` dependencies so I should be able to revert this.

Reply:
Done